raiden_contracts/tests/test_deploy_script.py

```diff
@@ -452,3 +453,12 @@ def test_error_removed_option_raises():
     with pytest.raises(NoSuchOption):
         mock = MagicMock()
         error_removed_option('msg')(None, mock, '0xaabbcc')
+
+
+def test_contracts_version_expects_deposit_limits():
+    assert not contracts_version_expects_deposit_limits('0.3._')
+    assert not contracts_version_expects_deposit_limits('0.4.0')
+    assert contracts_version_expects_deposit_limits('0.9.0')
+    assert contracts_version_expects_deposit_limits('0.10.0')
+    assert contracts_version_expects_deposit_limits('0.10.1')
+    assert contracts_version_expects_deposit_limits(None)
```

Comment:
Did you check the behavior for "illegal" `contracts_version` values?

Reply:
Good point.  I'll add some tests in this PR.